common/src/stack/report-system/command/report/system/tests/test_backend_link.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+import testinfra
+import paramiko
+import socket
+from stack import api
+
+
+testinfra_hosts = [host['host'] for host in api.Call('list host', args=['a:backend'])]
+class TestLinkUp:
+	""" See if the interfaces stacki configures are actually up on the hosts """
+
+	def test_link_status(self, host):
+		errors = []
+
+		# Get all interfaces for this host
+		try:
+			hostname = host.check_output('hostname')
+
+		except paramiko.ssh_exception.NoValidConnectionsError:
```

Comment:
Is this the actual exception raised by testinfra (I don't know, just making sure it is).

Reply:
That's what it raised when I was testing ssh to a host that wasn't up, it is a different exception than when using paramiko directly.